src/tiledb/cloud/utilities/_common.py

```diff
@@ -138,7 +140,11 @@ def run_dag(
             _print_logs(graph, debug=debug)
             raise
 
-    _print_logs(graph, debug=debug)
+    try:
+        _print_logs(graph, debug=debug)
+    except Exception:
+        # Ignore errors in case logs are not available.
+        pass
```

Comment:
Would it be worth it to do this in `_print_logs` itself?

Reply:
Sure, we can put it there in case we decide to call `_print_logs` somewhere else.